jobs/JEVS_AQM_PREP

```diff
@@ -73,6 +73,8 @@ export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/${evs_ver})}
 export DCOMIN=${DCOMIN:-${DCOMROOT}}
 export COMINaqm=${COMINaqm:-$(compath.py ${envir}/com/${modsys}/${mod_ver})}
 export COMOUT=${COMOUT:-$(compath.py ${envir}/com/${NET}/${evs_ver})}
+export COMOUTproc=${COMOUTproc:-${COMOUT}/${STEP}/${COMPONENT}/${RUN}.${VDATE}/${MODELNAME}}
```

Comment:
Hi @Ho-ChunHuang-NOAA , curious -- what does COMOUTproc mean in plain language? Maybe COMOUT for processed prep data?

Reply:
Yes.  for processing prep data out.

Reply:
I add a new environ because ${COMOUT}/${STEP}/${COMPONENT}/${RUN}.${VDATE}/${MODELNAME} has been repeatly used.

Reply:
> I add a new environ because ${COMOUT}/${STEP}/${COMPONENT}/${RUN}.${VDATE}/${MODELNAME} has been repeatly used.

Yeah, it is fine. It functions like COMOUTsmall and COMOUTfinal for stats or COMOUTplots for plots. Defining the one variable can minimize errors in repeating the string that you referenced above. It is good, I just wondered what "proc" was supposed to mean. Thanks!

Reply:
@ShelleyMelchior-NOAA You can use any name as you see it fit.  COMOUTprep or EVSOUT are some choices as well

Reply:
COMOUTproc works for me ... no more changes and we can merge the PR. ✔️